markdown/articles/2022/02/09/axionmegareport.md

```diff
@@ -0,0 +1,575 @@
+
+Anyways I was there to play and was determined to have fun so having met up with some of the rest of the Bristol
+crew, who were playing in the Unified Modern Trios event, and saying hello to our local judges, I headed
+to the player meeting. Francois gave a lovely speech and hinted at some "giga" events in the future
+which will be including Legacy in some form. Nik gave his official speech bit and we kicked off into
+round 1.
+
+
+#### Round 1 vs Connor on Yorion Death & Taxes
+
+We shuffle up and I lose the coin toss. Connor reveals a Yorion which immediately let's me work out
+he is likely on Death & Taxes or some sort of Blue Soup deck. I keep a 7 which has the potential for
+a turn 1 Doomsday. Connor goes :card[Plains] into :card[Aether Vial] and that pretty much
+lets me know the coast is clear. Annoyingly I could have had a turn 1 kill had I kept Deep Analysis
+in but instead I just do a clean pass-the-turn pile using Consider for a fast game 1. 
+
+Boarding is pretty simple for DnT, Massacre and Chain come in along with FoNs for :card[Deafening Silence].
+Discard, :card[CoS] and some :card[SW]/:card[Pre] come out.
+
+Game 2 showed the first of many misplays in the day that I blame the fatigue for. I keep 5 cards, another turn 1
+Doomsday with Force protection but no land (:card[DR] off of :card[LP]). Opponent once again tries to run
+out a Vial and I Force it pitching Force. This was a massive mistake. I should've worked out the timings
+of how many turns I would need to pass and what could beat me in passing that many turns. I would need
+to pass 3 turns post Doomsday (unless I get lucky on my draw step with a land or another Petal,:card[BS],
+Street Wraith or :card[LED]) and so I should've worked out what beats the likely pile of {fetch, fetch, TO,
+:card[Island], :card[Underground Sea]} or even what beats me trying to go for a faster 2 turn pass pile
+that maybe uses Brainstorm or something else similar. Basically there is not enough pressure that
+can come down from 2 lands and Vial for passing 3 turns assuming you hold Force for the first thing
+they cast. The Forcing of Vial was a poor decision and I should've just held it back for a turn 2 play.
+
+:::row{variant=pile}
+- Flooded Strand
+- Misty Rainforest
+- TO
+- Island
+- Underground Sea
+:::
+
+So I draw a dud for turn and jam Doomsday building exactly what was described above and pass the turn.
+I was justifiably punished by Connor jamming a turn 2 :card[Stoneforge Mystic] which promptly finds
+:card[Kaldra Compleat] and simple as that I was dead. I play land and say go, he puts Kaldra in and hits
+for 5. Land go. Hit for 5 and dead.
+
+We learn from out mistakes and move on. Game 3 I keep 7, Connor 5 and I once again jam a turn 1 Doomsday. 
+With a full grip and on the play there are very few scenarios where a Death & Taxes player can win
+from here and we get our first (somewhat sloppy) win of the day.
+
+```
+Match:   2-1
+Matches: 1-0
+Games:   2-1
+```
```

Comment:
That markup is reserved for code blocks and get special treatment so you might not want that here. See here for a example of what it looks like with code https://ddft.wiki/articles/2020/03/21/the-mindkiller#reality-check---a-bit-of-math-regarding-spoils

![image](https://user-images.githubusercontent.com/5608624/153182826-17cc137f-ff7f-48d6-802a-1b0b6d3a5561.png)

A table will look questionable here as well I think but feel free to try it. Alternatively, how about regular text with bold if you want it to stand out in each section 🤔

Reply:
Gone for the bold option